src/road_safety_scanner.py

```diff
@@ -161,13 +169,363 @@ def __init__(self: Self) -> None:
         header.setSectionResizeMode(QHeaderView.Stretch)
 
         self.uploadedJournals = []
-
-        # Connect the open file button to the openFile function
-        self.ui.uploadJournalButton.clicked.connect(self.open_file)
         
         # Connect the export button to the handleExport function
         self.ui.exportResultsButton.clicked.connect(self.handle_export)
 
+        # If sets.json does not exists then it creates an empty .json file
+        file_path = "src/modules/GUI/sets.json"
```

Comment:
The journal sets shouldn't be stored in GUI- ideally would make more sense to store them in journal_downloader.

Come to think of it, all this journal set code should probably be moved into a subfeature of journal_downloader.